src/moveit_commander/planning_scene_interface.py

```diff
@@ -105,6 +105,14 @@ def __make_mesh(self, name, pose, filename, scale = (1, 1, 1)):
         pyassimp.release(scene)
         return co
     
+    def __make_existing(self, name):
+        """
+        Create an empty Collision Object, used when the object already exists 
+        """
+        co = CollisionObject()
+        co.id = name
```

Comment:
i actually do not use moveit_commander but i would assume it already has a planning scene monitor initialized that is keeping track of all the planning scene objects. does it not?